provider/cmd/test_helpers.go

```diff
@@ -0,0 +1,33 @@
+package cmd
```

Comment:
put this in a file that ends in `_test.go` so that it's only included when testing.  or, ideally, in a separate test util package.

Reply:
That is a much better pattern. :rocket: